### ISSUE TITLE
Cove-shaped ST segment elevation on ECG

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10032]
**Preferred term label:**
Cove-shaped ST segment elevation on ECG

**Synonyms**
Coved type cove-shaped characteristically seen in the Brugada syndrome; includes type 1 or type 2 Brugada pattern; also described as J point elevation followed by a descending ST segment which descends into a negative T wave

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://www.ncbi.nlm.nih.gov/medgen/C5775315
https://www.ncbi.nlm.nih.gov/medgen/C5775319

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Brugada syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000984
- label: Coved type ST segment elevation
- parents: [ST segment elevation (HP:0012251)](https://hpo.jax.org/app/browse/term/HP:0012251)
- pmid: PMID:30139433
- def: An ST-segment elevation of at least 2 mm in at least one right precordial lead (V1 to V3), followed by an r'-wave and a concave or straight ST segment. The descending ST segment crosses the isoelectric line and is followed by a negative and symmetric T-wave.
- comment: This finding is highly suggestive of Brugada syndrome.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10032
- synonyms:
